_config.yml

```diff
@@ -1,9 +1,11 @@
 # ----
 # Site
 
-title: Hydra Template
-url: "https://tableuv.github.io"
-baseurl: "/"
+title: TableUV
+url:
+baseurl:
```

Comment:
```suggestion
baseurl: "/"
```

Reply:
When working locally, baseurl should be set as empty.
When pushing to github baseurl should be set to "/"